lib/msal-core/src/Constants.ts

```diff
@@ -80,6 +80,9 @@ export class Constants {
 
   static get cacheLocationLocal(): CacheLocation { return "localStorage"; }
   static get cacheLocationSession(): CacheLocation { return "sessionStorage"; }
+
+  static get interactionTypeRedirect(): InteractionType { return "redirectInteraction"; }
```

Comment:
Might be better to use an enum here 

```js
enum InteractionType {
    Redirect, 
    PopUp
}
```

Reply:
I will keep this consistent with cache location for now, update to enums in a future PR